pysparkling/context.py

```diff
@@ -23,7 +23,10 @@
 
 # Python 2 compatibility
```

Comment:
Should this whole block be removed now? Can also leave it for later cleanup.

Reply:
It is also possible to keep Python2 support in existing code, the effort to remove compatibility code does not seem worth it